### ISSUE TITLE
Create index of MSL mastcam

### ISSUE THREAD

**Issue by open768** [open768/spaceinc#3]
To be able to do a random image

**Comment by open768** [comment]
Will need an admin option

**Comment by open768** [comment]
Needs to be a batch process. Must remember state to be able to pick up if script is killed.

Already have reading functions, just need to be able to store in a new table

Last 5 sols should always be reprocessed as products are added.

**Comment by open768** [comment]
Table needs to have in plain text
- mission
- sol
- instr
- product

And those cold are unique

**Comment by open768** [comment]
code written :-) seems to work, just need to run to completion

**Comment by open768** [comment]
should use the last updated to reindex
```
    {
      "sol": 125,
      "num_images": 245,
      "catalog_url": "http://mars.jpl.nasa.gov/msl-raw-images/image/images_sol125.json",
      "last_updated": "2014-08-14T01:08:31Z"
    },
```

**Comment by open768** [comment]
last updated comment moved to #4

**Comment by open768** [comment]
added function to delete contents from indexing  - reindexed sols

**Comment by open768** [comment]
so the json files contains thumbnails which need to be stripped out

**Comment by open768** [comment]
fixed code in commit d3c68613d3a148993d9865cbd4c60f0bbe7ddced

**Comment by open768** [comment]
made indexing super fast by only checking for cache expiry in last 10 sols 98dfa23ede99f29733028e061e56c6e253d7fe2e
